components/logins/src/encryption.rs

```diff
@@ -36,6 +36,19 @@ pub struct EncryptorDecryptor {
     jwk: jwcrypto::Jwk,
 }
 
+trait CryptoResultExt<T> {
+    fn map_crypto_err(self, action: &str) -> Result<T>;
+}
+
+impl<T> CryptoResultExt<T> for std::result::Result<T, jwcrypto::JwCryptoError> {
+    fn map_crypto_err(self, action: &str) -> Result<T> {
+        self.map_err(|e| Error::CryptoError {
+            action: action.into(),
+            error: e,
+        })
+    }
+}
```

Comment:
What do you think about moving this to the `error` module?

Also, is this an abstraction credit cards can use too? if we feel like it is, we can move it to our shared `types` crate and have the `Error` be an associated type and implement it for both

But we can also just keep this here for logins if you feel like that would be pre-mature optimization!

Reply:
eeh I just realized moving this trait to another crate means we can't implement it for `std::result::Result` - never mind this comment - leaving it open for the "moving this to the `error` module part